app/editor/src/features/reports/av-overview/AVOverview.tsx

```diff
@@ -59,15 +72,23 @@ export const AVOverview: React.FC = () => {
     [api],
   );
 
+  const handlePublish = React.useCallback(async () => {
+    try {
+      const result = await api.publishAVOverview(instance.id);
+      toast.success('AV Evening Overview report request to publish has been sent.');
+      setInstance(result);
+    } catch {}
+  }, [api, instance.id]);
+
   return (
     <styled.AVOverview>
       <Row className="page-header">
-        <div className="title">AV Evening Overview</div>
+        <h1>AV Evening Overview</h1>
```

Comment:
why the switch to H1 here?  We don't seem to have used that tag much anywhere else in the codebase.

Reply:
We should be using them more than we are.  They are meant for this very purpose.  I haven't been as careful in expecting specific HTML or styling implementations.  I only catch so much in each PR and we're racing against the clock.

Reply:
I definitely agree that we should be using it more, I'm just worried about consistency.

Reply:
Help me catch these types of things in future PRs.  We technically should have proper styling setup globally for headers of each size.